renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": [
+    "config:base",
+    ":rebaseStalePrs",
+    ":preserveSemverRanges",
+    ":semanticPrefixChore",
+    ":disableRateLimiting"
+  ],
+  "gradle": {
+    "enabled": true
+  },
+  "renovateFork": true
```

Comment:
This allows renovate to be run on forks of uPortal

Reply:
Why would we want Renovate to look at forks? I'm not seeing the value of this ...

Reply:
No worries, it's a confusing option.